### PR TITLE
feature 2266: People page now gets settings from query parameters

### PR DESCRIPTION
**Pull request by mvolkmann**

### CODE REVIEW

File: web-ui/src/pages/PeoplePage.jsx [hunk 1]
```diff
@@ -42,6 +42,23 @@ const PeoplePage = () => {
 
   const normalizedMembers = selectNormalizedMembers(state, searchText);
 
+  useEffect(() => {
```
Comment (mvolkmann) [inline web-ui/src/pages/PeoplePage.jsx:45]: On page load this gets settings from query parameters.

File: web-ui/src/pages/PeoplePage.jsx [hunk 2]
```diff
@@ -42,6 +42,23 @@ const PeoplePage = () => {
...
+    const url = new URL(location.href);
+    const search = url.searchParams.get('search') || '';
+    setSearchText(search);
+  }, []);
+
+  useEffect(() => {
```
Comment (mvolkmann) [inline web-ui/src/pages/PeoplePage.jsx:51]: When settings change, this updates the URL query parameters.

File: web-ui/src/pages/PeoplePage.jsx [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import React, { useContext, useState } from 'react';
+import { TextField, Grid } from '@mui/material';
```
Comment (mvolkmann) [inline web-ui/src/pages/PeoplePage.jsx:2]: I reordered some of the imports.

File: web-ui/src/pages/PeoplePage.jsx [hunk 4]
```diff
@@ -42,6 +45,15 @@ const PeoplePage = () => {
 
   const normalizedMembers = selectNormalizedMembers(state, searchText);
 
+  useQueryParameters([
```
Comment (mvolkmann) [inline web-ui/src/pages/PeoplePage.jsx:48]: This is the main change.

**Review by S78901 — APPROVED** [review]

**Review by vhscom — APPROVED** [review]

**Pull request merged**